AsyncDisplayKit/ASTextNode.mm

```diff
@@ -250,9 +248,28 @@ - (ASTextKitAttributes)_rendererAttributes
   };
```

Comment:
Yeah, my opinion is to add a comment. Either that the lock is already held by the calling method but we're locking again for completeness and readability *or* don't call the lock and note above the method that it must be called locked. Would it make sense to adopt some type of method naming scheme for methods that must be called with the lock held?